clusterloader2/pkg/measurement/common/simple/etcd_metrics.go

```diff
@@ -83,9 +83,13 @@ func (e *etcdMetricsMeasurement) Execute(config *measurement.MeasurementConfig)
 		if err = e.stopAndSummarize(host, provider); err != nil {
 			return summaries, err
 		}
-		summaries := append(summaries, e.metrics)
+		content, err := util.PrettyPrintJSON(e.metrics)
+		if err != nil {
+			return summaries, err
+		}
+		summary := measurement.CreateSummary(etcdMetricsMetricName, "json", content)
+		summaries := append(summaries, summary)
```

Comment:
Given you're touching this code anyway, let's maybe change that.
Both here and in pretty much every other measurement, there is exactly 1 place where you return non-empty summaries. We should switch to:
- not define summaries variable at all
- return nil, if we return an error (all places but this)
- here simply to:
return []measurement.Summary{summary}, nil

Same in every other measurement, given you're touching it anyway.

Reply:
Done. (separate commit)